src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -14,7 +14,6 @@
    {@comments(fileHeader.licenseLines)}
    {@editingInstructions(protoFilename)}
    */
-  /* TODO: introduce line-wrapping so that it never exceeds the limit. */
   /* jscs: disable maximumLineLength */
```

Comment:
This line can go away too. We no longer use have.

Reply:
*jscs

Reply:
Done.